Description: Adjust description width in the netstat graphs
 The text below the ifstat RRD graphic is quite unreadable, because it
 shows all numbers as 10.1lf, which means you have very long numbers
 which don't fit into the graphic.  As an alternativ I suggest
 formatting the numbers as 5.1lf%s like the netstat uses, which looks
 much more readable.
Author: Roland Rosenfeld <roland@spinnaker.de>
Bug-Debian: https://bugs.debian.org/447484
Acked-By: Christoph Berg <myon@debian.org>
Last-Update: 2015-09-10
Forwarded: no

Index: xymon/xymond/etcfiles/graphs.cfg
===================================================================
--- xymon.orig/xymond/etcfiles/graphs.cfg	2014-01-10 01:59:49.574516438 +0100
+++ xymon/xymond/etcfiles/graphs.cfg	2014-01-10 02:00:09.770881261 +0100
@@ -1111,15 +1111,15 @@
 	DEF:outbytes@RRDIDX@=@RRDFN@:ds1:AVERAGE
 	CDEF:out@RRDIDX@=outbytes@RRDIDX@,8,*
 	LINE2:in@RRDIDX@#@COLOR@:@RRDPARAM@  inbound
-	GPRINT:in@RRDIDX@:LAST: \: %10.1lf (cur)
-	GPRINT:in@RRDIDX@:MAX: \: %10.1lf (max)
-	GPRINT:in@RRDIDX@:MIN: \: %10.1lf (min)
-	GPRINT:in@RRDIDX@:AVERAGE: \: %10.1lf (avg)\n
+	GPRINT:in@RRDIDX@:LAST: \: %5.1lf%s (cur)
+	GPRINT:in@RRDIDX@:MAX: \: %5.1lf%s (max)
+	GPRINT:in@RRDIDX@:MIN: \: %5.1lf%s (min)
+	GPRINT:in@RRDIDX@:AVERAGE: \: %5.1lf%s (avg)\n
 	LINE2:out@RRDIDX@#@COLOR@:@RRDPARAM@ outbound
-	GPRINT:out@RRDIDX@:LAST: \: %10.1lf (cur)
-	GPRINT:out@RRDIDX@:MAX: \: %10.1lf (max)
-	GPRINT:out@RRDIDX@:MIN: \: %10.1lf (min)
-	GPRINT:out@RRDIDX@:AVERAGE: \: %10.1lf (avg)\n
+	GPRINT:out@RRDIDX@:LAST: \: %5.1lf%s (cur)
+	GPRINT:out@RRDIDX@:MAX: \: %5.1lf%s (max)
+	GPRINT:out@RRDIDX@:MIN: \: %5.1lf%s (min)
+	GPRINT:out@RRDIDX@:AVERAGE: \: %5.1lf%s (avg)\n
 
 [mysqlslow]
 	DEF:slow=mysql.rrd:Slowqueries:AVERAGE
